yt/frontends/ramses/io.py

```diff
@@ -156,9 +159,66 @@ def _read_particle_subset(self, subset, fields):
 
             else:
                 # Raise here an exception
-                raise Exception('Unknown particle type %s' % ptype)
+                raise YTFieldTypeNotFound(ptype)
 
             tr.update(_ramses_particle_file_handler(
                 fname, foffsets, data_types, subset, subs_fields))
 
         return tr
+
+VERSION_RE = re.compile('# version: *(\d+)')
+VAR_DESC_RE = re.compile(r'\s*(\d+),\s*(\w+),\s*(\w+)')
```

Comment:
Are those anywhere except in `_read_part_file_descriptor`? In any case, could we move initialization out of global scope?